helm/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
My criteria has been to differ as little as possible from the skeleton provided by `helm create` but, if this is how we prefer it, I guess more things could/should be changed.